src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.NodeMapBuilder.cs

```diff
@@ -255,6 +255,19 @@ public override BoundNode VisitBinaryOperator(BoundBinaryOperator node)
                 throw ExceptionUtilities.Unreachable;
             }
 
+            public override BoundNode VisitDeconstructionAssignmentOperator(BoundDeconstructionAssignmentOperator node)
```

Comment:
>VisitDeconstructionAssignmentOperator [](start = 38, length = 37)

What is the reason for this?  I'm not as familiar with the node map so it's unclear why we have this behavior.